install/update.php

```diff
@@ -889,12 +956,29 @@ function pluginFusioninventoryUpdate($current_version, $migrationname = 'Migrati
    }
 
    // Fix software version in computers. see https://github.com/fusioninventory/fusioninventory-for-glpi/issues/1810
-   $query = "SELECT * FROM `glpi_computers` WHERE `entities_id` > 0";
-   $result=$DB->query($query);
-   while ($data=$DB->fetch_array($result)) {
-      $DB->query("UPDATE `glpi_computers_softwareversions` SET `entities_id`='".$data['entities_id']."'
-                     WHERE `computers_id`='".$data['id']."'"
-              . "AND `is_dynamic`='1' AND `entities_id`='0'");
+   $iterator = $DB->request([
+      'FROM'   => 'glpi_computers',
+      'WHERE'  => ['entities_id' => ['>', 0]]
+   ]);
+   if (count($iterator)) {
+      $update = $DB->buildUpdate(
+         'glpi_computers_softwareversions', [
+            'entities_id'  => new \QueryParam()
+         ], [
+            'computers_id' => new \QueryParam(),
+            'is_dynamic'   => 1
```

Comment:
it seems you miss a param here: entities_id => 0.
Correctly set in first param of ``bind_param`` function below.